src/schemas.jl

```diff
@@ -98,8 +98,12 @@ end
 
 struct UnknownSchemaVersionError <: Exception
     schema_version::SchemaVersion
+    schema_provider_name::Union{Missing, Symbol}
+    schema_provider_version::Union{Missing, VersionNumber}
```

Comment:
Looks like we don't have lint checking on this repo. I think the these don't have spaces
```suggestion
    schema_provider_name::Union{Missing,Symbol}
    schema_provider_version::Union{Missing,VersionNumber}
```